src/Dmishh/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -107,4 +110,20 @@ protected function manage(Request $request, SettingsOwnerInterface $owner = null
             )
         );
     }
+
+    /**
+     * Get SecurityContext service
+     * @param string $service The service name
+     *
+     * @return mixed The service
+     */
+    private function getSecurityContext($service)
```

Comment:
I understand why this is done. There are two use cases for this function

```php
$this->getSecurityContext('security.token_storage');
$this->getSecurityContext('security.authorization_checker');
```
In SF<2.6 we aways return security context. Sure, it might look weird. 

May I suggest creating two private functions? 
```php
getTokenStorage()
getAuthorizationChecker()
```

It make the code more readable and it will make more sense in the future.